tests.py

```diff
@@ -1,196 +1,149 @@
-            "JON234",
-            "JON234",
-            "JON234",
-            "RUMI874",
-            "RUMI874",
-            "RUMI874",
-            "JONNA465",
-            "JONNA465",
-            "JONNA465",
-            "JONNA465",
-        ],
-        "LabName": [
-            "METABOLIC: TOTAL PROTEIN",
-            "CBC: HEMATOCRIT",
-            "URINALYSIS: RED BLOOD CELLS",
-            "METABOLIC: GLUCOSE",
-            "URINALYSIS: RED BLOOD CELLS",
-            "CBC: HEMATOCRIT",
-            "METABOLIC: GLUCOSE",
-            "URINALYSIS: RED BLOOD CELLS",
-            "CBC: HEMATOCRIT",
-            "METABOLIC: TOTAL PROTEIN",
-            "URINALYSIS: RED BLOOD CELLS",
-            "CBC: HEMATOCRIT",
-            "METABOLIC: GLUCOSE",
-        ],
-        "LabValue": [
-            "4.2",
-            "3.5",
-            "10.8",
-            "10.0",
-            "12.4",
-            "9.9",
-            "23.0",
-            "3.8",
-            "18.3",
-            "14.0",
-            "15.4",
-            "13.1",
-            "12.3",
+    """Test Parse data function."""
+    labtest = "LABS TEST.txt"
+    pattest = "Patient test.txt"
+    anas = Patient(
+        "ANAS123",
+        "M",
+        "1996-04-04 02:45:40.547",
+        "Asian",
+        [
+            Lab(
+                "URINALYSIS: RED BLOOD CELLS",
+                "rbc/hpf",
+                "1.8",
+                "1999-07-01 01:36:17.910",
+            ),
+            Lab("METABOLIC: GLUCOSE", "mg/dL", "103.3", "1998-06-30 09:35:52.383"),
         ],
-        "LabDateTime": [
-            "2021-07-01 01:36:17.910",
-            "2022-01-01 01:36:17.910",
-            "2021-10-01 01:36:17.910",
-            "2004-07-01 01:36:17.910",
-            "2012-07-01 01:36:17.910",
-            "2014-07-01 01:36:17.910",
-            "2011-07-01 01:36:17.910",
-            "2012-07-01 01:36:17.910",
-            "2013-07-01 01:36:17.910",
-            "2011-07-01 01:36:17.910",
-            "2011-07-01 01:36:17.910",
-            "2011-07-01 01:36:17.910",
-            "2011-07-01 01:36:17.910",
+    )
+
+    jon = Patient(
+        "JON234",
+        "M",
+        "1976-12-28 02:45:40.547",
+        "White",
+        [
+            Lab("CBC: MCH", "pg", "35.8", "1992-06-30 03:50:11.777"),
+            Lab("METABOLIC: CALCIUM", "mg/dL", "8.9", "1992-06-30 12:09:46.107"),
         ],
-    }
+    )
 
-    assert parse_data(columns_only) == data_columns_only
-    assert parse_data(simple_data) == data_simple_parse
-    assert parse_data(patient_test_data) == parsed_patient_test
-    assert parse_data(labs_data_test) == parsed_patient_labs
+    answer = {"ANAS123": anas, "JON234": jon}
 
+    test = parse_data(labtest, pattest)
 
-# test num_older function
-def test_num_older():
-    """Test number older than a particular age."""
-    columns_only = "columns only.txt"
-    data_columns = parse_data(columns_only)
+    for i, j in zip(answer, test):
```

Comment:
You should be able to compare these more concisely. You may want to implement `__eq__()` on your classes.